src/styles/addOutlineWidth/types.ts

```diff
@@ -0,0 +1,6 @@
+import {CSSProperties} from 'react';
+import {TextStyle} from 'react-native';
+
+type AddOutlineWidth = (obj: TextStyle | CSSProperties, val: number, error: boolean) => TextStyle | CSSProperties;
```

Comment:
Shouldn't val and error be optional? @fabioh8010

Reply:
@blazejkustra Updated!